docs/install-azure-repos.md

```diff
@@ -0,0 +1,180 @@
+[Go back to the main documentation page](../README.md)
+# Installing Frogbot on Azure Repos repositories
+
+To install Frogbot on Azure Repos repositories:
+
+1. Go to Azure DevOps Pipelines of the relevant project, and add `New pipeline`.
+2. Set `Azure Repos Git` as your code source.
+3. Select the repository you'd like Frogbot to scan.
+4. Select 'Starter Pipeline' and use `frogbot-scan-pr.yml` and `frogbot-scan-and-fix.yml`:
+5. Make sure you have the connection details of your JFrog environment.
+6. In the pipeline page save the JFrog connection details as variables with the
+   following names - JF_URL, JF_USER, and JF_PASSWORD.
+
+> **_NOTE:_** You can also use **JF_XRAY_URL** and **JF_ARTIFACTORY_URL** instead of **JF_URL**, and **JF_ACCESS_TOKEN**
+> instead of **JF_USER** and **JF_PASSWORD**.
+
+7. To set variables in the pipeline page, click on `Variables` button and set `New variable`:
```

Comment:
It looks like section and sections 7 above should be mrged somehow, and the images should help the user understand when to find and set the variables.

Reply:
done